catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/content/ContentProducerDataAccessObject.java

```diff
@@ -192,25 +189,15 @@ public void createContentItem(FileSystemPersistenceProvider fileIdMap, ContentEn
 
     public void processHeaders(Map<String, Object> headers, StorageRequest storageRequest,
             File ingestedFile) {
-        String attributeOverrideHeaders = (String) headers.get(Constants.ATTRIBUTE_OVERRIDES_KEY);
-        if (StringUtils.isNotEmpty(attributeOverrideHeaders)) {
+        HashMap<String, Serializable> attributeOverrideHeaders = new HashMap<>((Map) headers.get(
```

Comment:
I believe Brendan is saying that he'd like the left side to be `Map<String, Serializable>`. As it's being used locally it's not super important to me, but in general I agree with that.